src/detext/run_detext.py

```diff
@@ -146,6 +146,12 @@ def add_arguments(parser):
 
     parser.add_argument('--use_horovod', type=str2bool, default=False, help="whether to use horovod for sync distributed training")
 
+    # multitask training related
+    parser.add_argument('--task_ids', type=str, default=None,
+                        help="All types of task IDs for multitask training. E.g., 1,2,3")
+    parser.add_argument('--task_weights', type=str, default=None,
```

Comment:
Suggest justifying why we need "task_weights" aside from "weight" in the tfrecord in the description. From my point of view, this addition is valid, but I do want to hear about your opinions

Reply:
This is a good question. I checked with detext users about this parameter. The weight feature is also valid for multitask training but it'll be more convenient to specify them as configurable parameters than features so that users do not need to regenerate training data when tuning the weight for each task.